beakerx/beakerx/static/tree.js

```diff
@@ -221,6 +214,22 @@ define(function(require) {
       }).val(value).keyup(InputChanged);
     },
 
+    normaliseHeapSize: function(val) {
+      if (val === '') {
+        return '';
+      }
+
+      var parsedVal = parseFloat(val);
```

Comment:
this is broken, please do not change how valid numbers are identified, it's actually quite tricky.
`parseFloat("5a") = 5`

Reply:
I did not change the way it was identified

see:
https://github.com/twosigma/beakerx/blob/27dab19f5c88a5e03cbb479a6da80a3bba020b73/beakerx/beakerx/static/tree.js#L54-L64
https://github.com/twosigma/beakerx/blob/27dab19f5c88a5e03cbb479a6da80a3bba020b73/beakerx/beakerx/static/tree.js#L98-L101

All i did with it was a little extraction refactor, because i needed that logic in two different places.

I added additional check 
- isFinite(val) will catch "5a"